lib/java-wrapper.ts

```diff
@@ -77,21 +78,31 @@ export async function getCallGraph(
     config.CALL_GRAPH_GENERATOR_URL,
     config.CALL_GRAPH_GENERATOR_CHECKSUM,
   );
-  const entrypoints = await getEntrypoints(targetPath);
+  const entrypoints = await timeIt('getEntrypoints', () =>
+    getEntrypoints(targetPath),
+  );
   if (!entrypoints.length) {
     throw new Error('No entrypoints found.');
   }
-  const javaCommandArgs = getJavaCommandArgs(classPath, jarPath, entrypoints);
+  const callgraphGenCommandArgs = getCallGraphGenCommandArgs(
+    classPath,
+    jarPath,
+    entrypoints,
+  );
   try {
     const [javaOutput, classPerJarMapping] = await Promise.all([
-      runJavaCommand(javaCommandArgs, targetPath),
-      getClassPerJarMapping(classPath),
+      timeIt('generateCallGraph', () =>
```

Comment:
Maybe better to remove the `Promise.all` there? I'm afraid the node event loop will mess the results...

Reply:
the `timeIt` function returns a promise, So it is probably safe to do it this way.

Reply:
Meant more from the measurement perspective...
If `runJavaCommand` takes super long the event loop will be blocked and `getllassPerJarMapping` will report higher duration.

Reply:
makes sense!